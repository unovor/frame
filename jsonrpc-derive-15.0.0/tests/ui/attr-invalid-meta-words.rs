use jsonrpc_derive::rpc;

#[rpc]
pub trait Rpc {
	/// Returns a protocol version
	#[rpc(name = "protocolVersion", Xmeta)]
	fn protocol_version(&self) -> Result<String>;
}

fn main() {}
