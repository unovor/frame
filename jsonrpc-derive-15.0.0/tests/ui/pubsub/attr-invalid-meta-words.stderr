error: Invalid attribute parameter(s): 'Xsubscribe'. Expected 'subscribe, unsubscribe'
  --> $DIR/attr-invalid-meta-words.rs:10:2
   |
10 |       /// Hello subscription
   |  _____^
11 | |     #[pubsub(subscription = "hello", Xsubscribe, name = "hello_subscribe", alias("hello_sub"))]
12 | |     fn subscribe(&self, _: Self::Metadata, _: typed::Subscriber<String>, _: u64);
   | |_________________________________________________________________________________^

error: aborting due to previous error

